wzdx/standard_to_enhanced/icone_translator.py

```diff
@@ -1,71 +1,79 @@
-
 import argparse
 import json
 import logging
 import copy
-from wzdx.sample_files.validation_schema import wzdx_v31_feed
-
-from wzdx.tools import date_tools, polygon_tools, wzdx_translator
-
-PROGRAM_NAME = 'IconeTranslator'
-PROGRAM_VERSION = '1.0'
+import uuid
+from ..sample_files.validation_schema import work_zone_feed_v42
 
+from ..tools import date_tools, wzdx_translator, geospatial_tools
 
-DEFAULT_ICONE_FEED_INFO_ID = '104d7746-688c-44ed-b195-2ee948bf9dfa'
+PROGRAM_NAME = "IconeTranslator"
+PROGRAM_VERSION = "1.0"
 
 
 def main():
     input_file, output_file = parse_icone_arguments()
     # Added encoding argument because of weird character at start of incidents.xml file
 
-    icone_obj = json.loads(open(input_file, 'r').read())
+    icone_obj = json.loads(open(input_file, "r").read())
     wzdx = wzdx_creator(icone_obj)
-    wzdx_schema = wzdx_v31_feed.wzdx_v31_schema_string
 
-    if not wzdx_translator.validate_wzdx(wzdx, wzdx_schema):
+    if not wzdx:
         logging.error(
-            'validation error more message are printed above. output file is not created because the message failed validation.')
+            "Generation error more message are printed above. output file is not created because the message failed validation."
+        )
         return
-    with open(output_file, 'w') as fwzdx:
-        fwzdx.write(json.dumps(wzdx, indent=2))
-        print('Your wzdx message was successfully generated and is located here: ' + str(output_file))
+    with open(output_file, "w") as fWzdx:
+        fWzdx.write(json.dumps(wzdx, indent=2))
+        print(
```

Comment:
Print to logging swap

Reply:
Updated other logging statements to use print